install/operator/pkg/syndesis/action/install.go

```diff
@@ -245,6 +260,37 @@ func (a *installAction) Execute(ctx context.Context, syndesis *v1alpha1.Syndesis
 	return err
 }
 
+// Checks that the tags from syndesis components is valid beetween the supported versions
+func checkTags(context *generator.Context) error {
+	c := version.NewConstrainGroupFromString(fmt.Sprintf(">=%s,<%s", context.TagMinor, context.TagMajor))
+	if c.Match(version.Normalize(context.Syndesis.Spec.Components.Server.Tag)) == false {
+		return fmt.Errorf("tag for server[%s] component is not supporter, versions supported are between [%s] and [%s]",
```

Comment:
I've noticed the typo, which made me thing that perhaps we can make this a bit clearer
```suggestion
		return fmt.Errorf("tag for server[%s] component is not valid, tag should have a value between [%s] (inclusive) and [%s] (exclusive)",
```

Similarly for the messages below.